libtenzir/builtins/operators/serve.cpp

```diff
@@ -103,6 +103,10 @@ constexpr auto SPEC_V0 = R"_(
                 type: integer
                 example: 50
                 description: The maximum number of events returned. If unset, the number is unlimited.
+              min_events:
+                type: integer
+                example: 50
+                description: Wait for this number of events before returning. If unset, the number is unlimited.
```

Comment:
Can you add an example to the `serve` operator documentation that uses the new parameter?